test/event/staking/staking-factory-events.ts

```diff
@@ -10,13 +10,18 @@ type RewardToken = {
 
 export type ActualStakingPoolCreatedEvent = {
     stakingPool: string
-    treasury: string
     creator: string
+    config: Config
+}
+type Config = {
+    treasury: string
     rewardTokens: RewardToken[]
     stakeToken: string
-    epochStartTimestamp: BigNumber
-    epochDuration: BigNumber
+    epochStartTimestamp: number
```

Comment:
question: why the change to type number?

_There are now both `number` and `BigNumber` being used_

Reply:
These are now both BigNumber's. I think it's simpler this way.

Reply:
This PR definitely is showing them as being added as `number`

Reply:
I think somehow (?!) this was stale